composer.json

```diff
@@ -90,8 +90,8 @@
         "phpspec/php-diff": "^1.0",
         "phpunit/phpunit": "^9.5.19",
         "psr/cache": "^1.0 || ^2.0 || ^3.0",
-        "psr/container": "^1.0",
-        "psr/http-message": "^1.0",
+        "psr/container": "^1.0 || ^2.0",
+        "psr/http-message": "^1.0 || ^2.0",
```

Comment:
@ausi Does it make sense to keep all the old versions here? For example, `psr/container` version 1 is not installable with `symfony/service-contracts` 3 anyway.

Reply:
If it is not installable anyways, I’d remove the old version then to keep it clear and simple.

```json
        "psr/container": "^2.0",
        "psr/http-message": "^2.0",
```